.github/workflows/humble-debian-build.yml

```diff
@@ -0,0 +1,33 @@
+name: Humble debian Build
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - humble
+  pull_request:
+    branches:
+      - humble
+  schedule:
+    # Run every day to detect flakiness and broken dependencies
+    - cron: '03 1 * * *'
+
+
+jobs:
+  humble_debian:
+    name: Humble debian build
+    runs-on: ubuntu-latest
+    env:
+      ROS_DISTRO: humble
+    container: ghcr.io/christophfroehlich/ros:humble-debian
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: src/ros2_control
+      - name: Build and test
+        shell: bash
+        run: |
+          source /opt/ros2_ws/install/setup.bash
+          vcs import --input https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/ros2_control.${{ env.ROS_DISTRO }}.repos?token=${{ secrets.GITHUB_TOKEN }} src
```

Comment:
do we need to keep the humble repos file up to date on the master branch with this?

Reply:
Good point! We can just use the repos file from the file system instead, it got already checked out